projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -513,6 +513,7 @@ class="screen-reader-text"
 							: 'width: 95%; padding: 1px 10px'
 						),
 						esc_attr( $data['subscribe_placeholder'] ),
+						esc_attr( $data['subscribe_email'] ),
```

Comment:
Is the array key reliable enough? Should we check isset/empty?

Reply:
Following the pattern here no element of the array is checked cause the caller is making sure it's set